### DISCUSSION TITLE
Stuck at specifying tag/release for git URL.

### DISCUSSION THREAD

**Discussion by acagastya** [Homebrew/discussions#2995] [Writing Formulae/Casks] [answered]
Hi, I am trying to submit a formulæ, and url I've specified is `https://git.hanabi.in/repos/wordle-cli.git`.  When I do `brew audit --strict --online formulæ`, I get:
```
  * 4: col 3: Formulae in homebrew/core should specify a revision for git URLs
  * 4: col 3: Formulae in homebrew/core should specify a tag for git URLs
```

Any idea how I fix that?

**Comment by SMillerDev** [comment, marked as answer]
Specify a revision and a tag?

See https://docs.brew.sh/Formula-Cookbook#unstable-versions-head for the functions you can use there. or search the homebrew/core repository for prior art.

**Comment by carlocab** [comment]
Additionally, please run `brew audit --new` instead of `brew audit --strict` for new formulae.
